docs/sdks/client-side-identity.md

```diff
@@ -232,11 +244,11 @@ All interactions with the Client-Side JavaScript SDK are done through the global
 - [abort()](#abort-void)
 - [callbacks](#callbacks) <New />
 - [setIdentity()](#setidentityidentity-uid2identity-void) <New />
-- [getIdentity()](TODO) <New />
+- [getIdentity()](#getidentity-uid2identity--null) <New />
 
 ### constructor()
 
-Constructs a UID2 object. This is not intended to be used directly: when the SDK loads, it automatically initializes an instance of the UID2 class and stores it as the global `__uid2` object. Advanced integrations may make use of this constructor directly, but care must be taken to avoid having multiple active instances of the SDK running. This is not a supported use-case.
+Constructs a UID2 object. This is not intended to be used directly: when the SDK loads, it automatically initializes an instance of the UID2 class and stores it as the global `__uid2` object. Advanced integrations could make use of this constructor directly, but be careful to avoid having multiple active instances of the SDK running. This is not a supported use case.
```

Comment:
I'd like to keep using the words `may` and `must` here because they have special meaning in software engineering (see https://www.ietf.org/rfc/rfc2119.txt for details - while that document is specific to the meaning in IETF documents, the IETF is the tech industry's main standards body so the meaning is generally considered universal in tech industry documents).

Perhaps:

> Constructs a UID2 object. This is not intended to be used directly: when the SDK loads, it automatically initializes an instance of the UID2 class and stores it as the global `__uid2` object. Advanced integrations may make use of this constructor directly, but must take care to avoid having multiple active instances of the SDK running. This is not a supported use case.

Reply:
Done thx! And good to know. Generally I try not to use "may" because it's ambiguous -- it can express possibility (it may rain today -- I use "might") or permission (as an administrator you may add a user -- I use "can"). I think I picked the wrong one here. But anyway it now says exactly what you suggested. Thx much.